lib/write.js

```diff
@@ -146,14 +146,7 @@ exports.usingStreaming = (deviceFileDescriptor, options = {}) => {
           transferredBytes += chunk.length;
 
           if (chunk.length < options.chunkSize) {
-            const finalChunkSize = _.attempt(() => {
-              if (chunk.length > options.minimumChunkSize) {
-                return options.chunkSize;
-              }
-
-              return options.minimumChunkSize;
-            });
-
+            const finalChunkSize = Math.ceil(chunk.length / options.minimumChunkSize) * options.minimumChunkSize;
```

Comment:
I'm not sure I follow.
If the chunk length were `127`, then above would yield `Math.ceil( 127 / 512 ) * 512)`, where the `Math.ceil()` would return `1` as `127 / 512 = ~0.248`, and the chunk would still be padded to `512` bytes.